model/http/deprecated/registry-deprecated.yaml

```
@@ -8,72 +8,97 @@ groups:
         type: string
         brief: 'Deprecated, use `http.request.method` instead.'
         stability: development
-        deprecated: "Replaced by `http.request.method`."
+        deprecated:
+          reason: renamed
+          renamed_to: http.request.method
         examples: ["GET", "POST", "HEAD"]
       - id: http.status_code
         type: int
         brief: 'Deprecated, use `http.response.status_code` instead.'
         stability: development
-        deprecated: "Replaced by `http.response.status_code`."
+        deprecated:
+          reason: renamed
+          renamed_to: http.response.status_code
         examples: [200]
       - id: http.scheme
         type: string
         brief: 'Deprecated, use `url.scheme` instead.'
         stability: development
-        deprecated: "Replaced by `url.scheme` instead."
+        deprecated:
+          reason: renamed
+          renamed_to: url.scheme
         examples: ['http', 'https']
       - id: http.url
         type: string
         brief: 'Deprecated, use `url.full` instead.'
         stability: development
-        deprecated: "Replaced by `url.full`."
+        deprecated:
+          reason: renamed
+          renamed_to: url.full
         examples: ['https://www.foo.bar/search?q=OpenTelemetry#SemConv']
       - id: http.target
         type: string
         brief: 'Deprecated, use `url.path` and `url.query` instead.'
         stability: development
-        deprecated: "Split to `url.path` and `url.query."
+        deprecated:
+          reason: obsoleted
+          note: Split to `url.path` and `url.query.
         examples: ['/search?q=OpenTelemetry#SemConv']
       - id: http.request_content_length
         type: int
         brief: 'Deprecated, use `http.request.header.<key>` instead.'
         stability: development
-        deprecated: "Replaced by `http.request.header.<key>`."
+        deprecated:
+          reason: uncategorized
+          note: Replaced by `http.request.header.<key>`.
```

wdyt? (and if so would want to change the brief above as well)
```suggestion
          note: Replaced by `http.request.header.content_length`.
```

---

I like it!